internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -557,6 +559,38 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 								},
 							},
 						},
+
+						"nat_gateway_profile": {
```

Comment:
i think we can remove profile from this property name?
```suggestion
						"nat_gateway": {
```

Reply:
Hi @katbyte! The naming is in this case based on the API property and consistent with existing property `load_balancer_profile`. I don't mind removing it though, just not sure.

Reply:
ahh yea given that profile is used for a bunch here this is fine even thou i don't like it 😬 to popular of resource to really rename everything here to clean it up and make it simplier/better 

fwiw we try to be more clear/simplier with property names over following the exact API names as they can often be... quite verbrose and strangely worded